Services/AbstractExporter.php

```diff
@@ -75,7 +75,7 @@ public function getDateTime(string $value = null, string $default = null): ?Carb
     {
         $value ??= $default;
 
-        return null !== $value ? $this->dateTimeHelper->parseUserDateTime($value)->setToDbTimezone() : null;
+        return null !== $value ? $this->dateTimeHelper->parseUserDateTime($value)->setToDbTimezone() : null; // @phpstan-ignore-line
```

Comment:
Consider using `@phpstan-ignore` (https://phpstan.org/user-guide/ignoring-errors#ignoring-in-code-using-phpdocs) here to ignore a specific error and add a comment on why it's ignored.

Reply:
I did not know you could do that. I will.